manifest.json

```diff
@@ -8,7 +8,7 @@
     },
     "url": "https://www.wallabag.org",
     "license": "MIT",
-    "version": "2.3.2-1",
+    "version": "2.3.7",
```

Comment:
```suggestion
    "version": "2.3.7~ynh1",
```

Reply:
What does that change ?

Reply:
You respect [official packaging rules](https://github.com/YunoHost/doc/blob/master/packaging_apps_manifest.md) :stuck_out_tongue:

Reply:
Don't forget to commit that change :wink: